apps/wallet-mobile/package.json

```diff
@@ -250,6 +250,10 @@
     "typescript": "4.8.4",
     "typescript-coverage-report": "^0.6.4"
   },
+  "resolutions": {
+    "@types/react": "18.2.0",
+    "@types/react-native": "0.71.6"
+  },
```

Comment:
@stackchain this needs to be added to the root `package.json` in the branch `chore/workspace-metrics`